lib/jellyfish_web/controllers/room_controller.ex

```diff
@@ -76,18 +80,18 @@ defmodule JellyfishWeb.RoomController do
   end
 
   def show(conn, %{"room_id" => id}) do
-    case RoomService.find_room(id) do
-      {:ok, room_pid} ->
-        room =
-          room_pid
-          |> Room.get_state()
-          |> maps_to_lists()
+    with {:ok, room_pid} <- RoomService.find_room(id),
+         true <- Process.alive?(room_pid) do
```

Comment:
This doesn't totally prevent calling a dead pid. IMO if you want to safely call `get_state` I'd add handling of `:noproc` error around `GenServer.call`